docs/INSTALL-GUIDE-LINUX.md

```diff
@@ -0,0 +1,84 @@
+# Linux support
+
+This tutorial uses Ubuntu 20.04.
+
+Unless specified otherwise, all linux distributions are supported.
+
+## 1. install node.js and npm
+
+It is recommend to use [nvm (Node Version Manager)](https://github.com/nvm-sh/nvm#install--update-script),
+ use this link to install latest nvm version.
+
+For example:
+
+```Shell
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+```
+
+note: in the past we used to use `apt` for these instructions, but on ubuntu 20.04 the default
+ package provided is outdated, plus it creates broken dependencies when using curl to get latest
+ version from node.js's github.
+
+Then update your shell environment (for example with Ubuntu 20.04 in this example):
+
+```Shell
+source ~/.profile
+```
+
+Display available versions to install:
+
+```Shell
+nvm ls-remote
+```
+
+Install the node version you want, for example to install node 14.2.0.0:
+
+```Shell
+nvm install 14.2.0
+```
+
+see for details [the full tutorial's linux node install page](https://github.com/wonderingabout/gtp2ogs-tutorial/blob/master/docs/3A1-linux-download-install-nodejs.md).
+
+## 2. install gtp2ogs using npm
+
+Install gtp2ogs locally, for example at the root of you user account,
+ create a new folder `gtp2ogs-node`.
+
+```Shell
+mkdir ~/gtp2ogs-node
+```
+
+Then go inside this folder and install gtp2ogs with npm:
+
+```Shell
+cd ~/gtp2ogs-node
+npm install gtp2ogs
+```
+
+gtp2ogs will be installed in `~/gtp2ogs-node/node_modules/gtp2ogs`
+
+## 3. Recommended: Upgrade to devel branch
```

Comment:
Why first download old gtp2ogs if you want to switch to dev anyway? I would skip step 2 if you are planing to do step 3 anyway.
That would also eliminate the need to delete the `node_modules/gtp2ogs` folder at this step.

Reply:
yes!!!!

but how do you install latest devel with npm ?
(for linux and windows)